src/Common/HashTable/Hash.h

```diff
@@ -89,6 +93,8 @@ inline DB::UInt64 intHashCRC32(DB::UInt64 x)
     return __crc32cd(-1U, x);
 #elif defined(__s390x__) && __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
     return s390x_crc32(-1U, x)
+#elif (defined(__PPC64__) || defined(__powerpc64__)) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
Cosmetic: Let's switch l. 94-95 and l. 96-97. (PPC is probably more important going forward than s390x). Same applies to the other files you changed.